ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -248,17 +257,20 @@ export class RemoteEnvironmentService extends EnvironmentService {
             }
             this.environmentExecutorManagerMap.set(environment.id, executorManager);
             const executor = await this.getExecutor(environment.id);
+            let remoteWorkingRoot: string;
```

Comment:
remoteWorkingRoot -> remoteWorkingRootDir

Reply:
modify it